README.md

```diff
@@ -1,70 +1,74 @@
+## Built With
 
-### `npm run build`
+- Create React App (CRA)
+- React Router
+- Gitflow
+- Redux toolkit
+- Jest test and snapshots
+- Coinstats API
+- React Hooks
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+## Getting Started
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+### Clone this repository
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+```bash
+$ git clone git@github.com:Favourezeugwa/Crypto-Metrics-Webapp.git
+$ cd math-magicians
+```
 
-### `npm run eject`
+### Run project
 
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+```bash
+$ npm install
+$ npm start
+```
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+### Open page in browser
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+```bash
+$ http://localhost:3001/
+```
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+## Author
 
-## Learn More
+👤 **Favour Amarachi Ezeugwa**
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+- GitHub: [@Favourezeugwa](https://github.com/Favourezeugwa)
+- LinkedIn: [Favour Amarachi Ezeugwa](https://www.linkedin.com/in/favour-amarachi-ezeugwa-a5bb31149/)
+- Twitter:[@Favour_ezeugwa](https://twitter.com/Favour_ezeugwa)
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+## 🤝 Contributing
 
-### Code Splitting
+Contributions, issues, and feature requests are welcome!
+Feel free to check the [issues page](https://github.com/Favourezeugwa/Crypto-Metrics-Webapp/issues)
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/code-splitting](https://facebook.github.io/create-react-app/docs/code-splitting)
+## Show your support
 
-### Analyzing the Bundle Size
+Give a ⭐️ if you like this project!
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
+## Acknowledgments
 
-### Making a Progressive Web App
+- Inspiration(Microverse Math magicians Wireframe)
+- Microverse student community
 
```

Comment:
- [ ] Kindly include an acknowledgment for Nelson Sakwa, the design author,

Reply:
Thank you @DammyShittu!